server/config/config.go

```diff
@@ -43,10 +43,11 @@ const (
 	defaultMinScanLimit    int  = 20
 	defaultIDAllocatorStep uint = 20
 
-	defaultClusterName              = "defaultCluster"
-	defaultClusterNodeCount         = 2
-	defaultClusterReplicationFactor = 1
-	defaultClusterShardTotal        = 8
+	defaultClusterName                       = "defaultCluster"
+	defaultClusterNodeCount                  = 2
+	defaultClusterReplicationFactor          = 1
+	defaultClusterShardTotal                 = 8
+	defaultClusterPartitionTableRatioOfNodes = 2
```

Comment:
The type should be float?

Reply:
I think int val maybe enough, since the type of nodes numbers is int, is it enough for us to use int radio directly? The use of float will not improve the accuracy.